core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -1316,13 +1316,14 @@ public void run() {
             try {
               serviceConfigInterceptor.handleUpdate(serviceConfig);
               if (retryEnabled) {
-                throttle = getThrottle(config);
+                throttle = ServiceConfigUtil.getThrottlePolicy(
+                    config.get(GrpcAttributes.NAME_RESOLVER_SERVICE_CONFIG));
```

Comment:
This can be simply `throttle = ServiceConfigUtil.getThrottlePolicy(serviceConfig)`

Reply:
Done.